packages/components-js/projects/partials/scripts/generateInitialStylesPartial.ts

```diff
@@ -8,19 +11,79 @@ export const generateInitialStylesPartial = (): string => {
   prefix?: string;
   ${withoutTagsOption}
   format?: Format;
+  // Standard normalize.css styles won't be applied by using this option
+  applyWithNormalizeStyles?: boolean;
 };
 type GetInitialStylesOptionsFormatHtml = Omit<GetInitialStylesOptions, 'withoutTags'> & { format: 'html' };
 type GetInitialStylesOptionsFormatJsx = Omit<GetInitialStylesOptions, 'withoutTags'> & { format: 'jsx' };
 type GetInitialStylesOptionsWithoutTags = Omit<GetInitialStylesOptions, 'format'>;`;
 
+  const normalizeStyles: Styles = {
+    '@global': {
+      '*': {
+        fontFamily: fontFamily,
+        lineHeight: fontLineHeight,
+        ...fontBehavior,
+      },
+
+      p: {
+        fontWeight: fontWeight.regular,
+        ...fontHyphenation,
+      },
+
+      'h1, h2, h3, h4, h5, h6': {
+        fontWeight: fontWeight.regular,
+      },
+
+      a: {
+        color: 'inherit',
+        textDecoration: 'underline',
+        '@media(hover:hover)': {
+          transition: 'color var(--p-transition-duration, .24s) ease',
+          '&:hover': {
+            color: themeLight.state.hover,
+          },
+        },
+      },
+
+      '[data-theme="dark"] a:hover': {
+        '@media(hover:hover)': {
+          '&:hover': {
```

Comment:
This produces `[data-theme="dark"] a:hover:hover{color:#ff0223}` which doesn't seem right.